### PR TITLE
docs: correct the object-deduplication design after review

### PR DESCRIPTION
**Pull request by richburdon**
Docs-only follow-up to #12417. That PR entered the merge queue while the project checkpoint was being written, so the push bounced off the protected branch and it merged with a design doc describing the pre-review implementation.

What `main` currently gets wrong:

- **The identity specs are contributed by `plugin-inbox`, not `plugin-crm`.** They moved during the PR: whoever creates the objects owns the rule for deciding when two of them are the same, and these are materialised from mail. The practical half matters as much — `plugin-crm` is a **Labs** plugin, so gating the Duplicates tab on a capability it contributes hid the review from every profile that had not enabled it. That is the reference most likely to mislead someone.
- The operation is `MergeDuplicates`, not `ApplyMerge`.
- The merge section predated four rounds of review. It now records what the code actually does: a confirmed draft is **assigned** over the merged result rather than folded through `spec.merge` (which only fills gaps, so an edit to a field the survivor already had was discarded); a value that cannot be normalized is **kept**, since the merge deletes the losers and anything skipped is destroyed; compound values are copied before joining the survivor because ECHO refuses to re-parent a nested record; uncommitted objects never enter the space-wide index; and `MergeDuplicates` re-derives the group before writing, because `planMerge` does not check identity — it only sorts and folds.

Also promotes the play test for the `Duplicates` story from a footnote to its own task. Every bug those four review rounds found was in the callers rather than the engine, and that is exactly the code with no automated coverage.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Documentation**
  * Clarified duplicate-management behavior, including merge handling, validation, draft preservation, and unnormalizable values.
  * Updated ownership and workflow guidance for the Duplicates experience.
  * Documented overlay index handling and state preservation during processing.
  * Recorded the completed walkthrough and identified follow-up automated test coverage.
* **Chores**
  * Updated project tracking records to reflect completed work, resolved reviews, verified fixes, and follow-up items.

### CODE REVIEW

File: .agents/projects/object-deduplication/TASKS.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Design: [DESIGN.md](./DESIGN.md). Branch `claude/person-deduplication-plugin-inb
 - [x] 1.6 Unit suite for Person duplicates — 21 tests green.
 - [x] 1.7 `IdentityIndex` + `makeIdentityIndex`/`buildIdentityIndex`.
 
-## Phase 2 — operations ✅
+## Phase 2 — operations ✅ (specs contributed by plugin-inbox — see DESIGN §3)
```
[the remark this answers is not in the document]
Comment (richburdon) [reply .agents/projects/object-deduplication/TASKS.md:15]: Fixed — and it was in the very PR correcting that reference elsewhere. Task 1.4 was stale the same way (`applyMerge` assigns overrides, it no longer folds them), so that is corrected too. The one remaining plugin-crm mention in DESIGN.md is deliberate: it explains why the specs moved off a Labs plugin.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
